devsetup/Makefile

```diff
@@ -175,10 +175,13 @@ edpm_deploy_instance: ## Spin a instance on edpm node
 edpm_play_cleanup: ## Cleanup EDPM openstackansibleee resource
 	-oc delete openstackansibleee deploy-external-dataplane-compute
 
-.PHONY: cifmw_prepare
-cifmw_prepare: ## Clone the ci-framework repository in the ci-framework directory. That location is ignored from git.
+./ci-framework:
```

Comment:
names ;) 
i'd call this something more descriptive though, like ci-framework-clone or somesuch.

Reply:
Oh yep, i would guess the aim was to also invoke the setup part, not sure - with that (renaming...) I would prefer to wait for Cedric.

Reply:
maybe we can add some more "checks" in there in order to ensure we can consume the framework - for instance "is ansible present" and related? or, even, run an actual task that will create the venv and make things ready to consume?

I really added that one in the very-very early stages of the framework, just to show "hey, we're here now". We can of course evolve (actually, we must evolve ;)).

Reply:
So what would be the recommendation for this PR please?
Rename (but that would involve changing all the docs / local flows too)? Or add more setup steps (any specific ones to add, or I can pick any)? Or leave it be as is?

Reply:
thanks for discussion today lets get this merged and you can update for more functionality as needed (imo)